### ISSUE TITLE
TypeError: "eval is not a valid URL." Policy.js:87:130

### ISSUE THREAD

**Issue by rhendric** [hackademix/noscript#100]
Whenever I navigate or switch to a tab that is not trusted, I see a few more copies of this error in the Browser Content Toolbox's console.

Here are a few representative stack traces:
```
 eval is not a valid URL. origin@moz-extension://3a1cc27e-fedc-449c-8c81-761012c230cd/common/Policy.js:87:130
requestKey@moz-extension://3a1cc27e-fedc-449c-8c81-761012c230cd/common/Policy.js:395:60
reportTo@moz-extension://3a1cc27e-fedc-449c-8c81-761012c230cd/bg/RequestGuard.js:226:23
onViolationReport@moz-extension://3a1cc27e-fedc-449c-8c81-761012c230cd/bg/RequestGuard.js:441:17
```

```
 eval is not a valid URL. origin@moz-extension://3a1cc27e-fedc-449c-8c81-761012c230cd/common/Policy.js:87:130
requestKey@moz-extension://3a1cc27e-fedc-449c-8c81-761012c230cd/common/Policy.js:395:60
_record@moz-extension://3a1cc27e-fedc-449c-8c81-761012c230cd/bg/RequestGuard.js:46:31
record@moz-extension://3a1cc27e-fedc-449c-8c81-761012c230cd/bg/RequestGuard.js:78:26
onViolationReport@moz-extension://3a1cc27e-fedc-449c-8c81-761012c230cd/bg/RequestGuard.js:442:19
```

```
 eval is not a valid URL. origin@moz-extension://3a1cc27e-fedc-449c-8c81-761012c230cd/common/Policy.js:87:130
requestKey@moz-extension://3a1cc27e-fedc-449c-8c81-761012c230cd/common/Policy.js:395:60
_record@moz-extension://3a1cc27e-fedc-449c-8c81-761012c230cd/bg/RequestGuard.js:46:31
recordAll@moz-extension://3a1cc27e-fedc-449c-8c81-761012c230cd/bg/RequestGuard.js:148:21
onActivatedTab@moz-extension://3a1cc27e-fedc-449c-8c81-761012c230cd/bg/RequestGuard.js:160:17
```

I'm running NoScript 11.0.3 in Firefox Developer Edition 69.0b13.

**Comment by hackademix** [comment]
Fixed by commit f9f116e.

**Comment by rhendric** [comment]
Confirmed fixed with NoScript 11.0.4rc3. Thank you!
